openmetadata-ui/src/main/resources/ui/src/components/AddDataQualityTest/components/TestCaseForm.tsx

```diff
@@ -357,6 +358,7 @@ const TestCaseForm: React.FC<TestCaseFormProps> = ({
             value: suite.fullyQualifiedName,
           }))}
           placeholder={t('label.select-field', { field: t('label.test-type') })}
+          onChange={handleTestDefinitionChange}
```

Comment:
onChange was added for data-testid="column" select but you changed it to data-testid="test-type"
any reason?

Reply:
We are showing the description of the test definition which is test type, seems like it was mistakenly added to the column